### ISSUE TITLE
I would like to contribute

### ISSUE THREAD

**Issue by andressa** [jminuscula/dixit-online#3]
Hi. First of all, congrats for the great idea about developing this!

I would be very interested in contributing wirh this project...

**Comment by jminuscula** [comment]
Hey @andressa, thanks a lot for the interest! I've just recently picked up on the project again, and have started upgrading the client code. As soon as I get it up and running it'll be ready for some contributions!

**Comment by sw-tracker** [comment]
Hi there, I would also love to contribute :)

**Comment by benkuly** [comment]
@andressa @sw-tracker A Dockerfile would be cool to better run it locally. Didn't really get it running, because of too old dependencies and my missing know how in python.

**Comment by Otto-AA** [comment]
@jminuscula How is it going? When the project is open for contributions, I could help a bit with the angular client.

**Comment by jminuscula** [comment]
@Otto-AA hey, thanks a lot for the interest! At the moment I'm trying to bring things up to date before I can kickstart the client development. So happy to get contributions once I get this walking ⚡

**Comment by rcjsuen** [comment]
> Didn't really get it running, because of too old dependencies and my missing know how in python.

@benkuly Yeah, I couldn't get `npm install` going either. I don't think the Python server will start right now based on what's currently in `master` (e7a1c9b5b1a2a92160c6d90397adaa81ddcae91a). I've opened #5 which should hopefully fix this initial roadblock.
